packages/form/addon/components/cf-field/input/checkbox.hbs

```diff
@@ -1,22 +1,28 @@
-{{#each @field.options as |option i|}}
-  {{#if (gt i 0)}}<br />{{/if}}
-  <label class="cf-checkbox_label {{if @field.isInvalid 'uk-form-danger'}}">
-    <input
-      class="uk-checkbox uk-margin-small-right"
-      type="checkbox"
-      name={{@field.pk}}
-      value={{option.slug}}
-      checked={{includes option.slug @field.answer.value}}
-      disabled={{@disabled}}
-      {{on "change" this.update}}
-    />
-    {{#if (and option.disabled (not @disabled))}}
-      <del
-        class="uk-text-muted"
-        title={{t "caluma.form.optionNotAvailable"}}
-      >{{option.label}}</del>
-    {{else}}
-      {{option.label}}
-    {{/if}}
-  </label>
-{{/each}}
\ No newline at end of file
+<div class={{if this.displayVertical "uk-flex"}}>
```

Comment:
Please put this class here: https://github.com/projectcaluma/ember-caluma/blob/c90c5e72dd300899f2d209216d0b893968ce6b20/packages/form/addon/components/cf-field/input.hbs#L3

This way we don't add an unnecessary div. Make sure that it's only applied for checkboxes and radio groups. You can do this by using the `hasQuestionType` helper. E.g.:

```hbs
<div class="uk-form-controls {{if (and (has-question-type @field.question "checkbox" "radio-group") @field.question.meta.vertical) 'uk-flex'}}"> 
```

Reply:
Done.